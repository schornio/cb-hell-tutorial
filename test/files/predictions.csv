Name; Status; isAzorAhai;
Tyrion; alive; false;
Jon; alive; true;
Stanis; alive; false;
Daenerys; alive; true;
