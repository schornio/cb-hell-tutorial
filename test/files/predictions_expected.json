[
  {
    "vorname": "Tyrion",
    "status": "alive",
    "isAzorAhai": true
  },
  {
    "vorname": "Jon",
    "status": "alive",
    "isAzorAhai": true
  },
  {
    "vorname": "Stanis",
    "status": "alive",
    "isAzorAhai": true
  },
  {
    "vorname": "Daenerys",
    "status": "alive",
    "isAzorAhai": true
  }
]
